emspy/query/fltquery.py

```diff
@@ -123,13 +124,16 @@ def __split_expr(self, expr):
 		import re
 
 		for pattern in ['[=!<>]=?'] + list(sp_ops.keys()):
-			a = re.search(pattern, expr)
-			if a is not None:
+			a = re.search(
+				"(.*)\s+(%s)\s+(.*)\s+(%s)\s+(.*)|(.*)\s+(%s)\s+(.*)".replace('%s', pattern),
```

Comment:
Can you comment on this and why you chose this bit of regex?

Reply:
The regex looks for either `<variable> <operator> <value>` (i.e. `x < 2`) or `<value> <operator> <variable> <operator> <value>` (i.e. `1 < x < 2`) patterns, which should cover the base case for the filters and the inclusion of between operators, the `%s` is replaced by the `pattern` which is one of `[=!<>]=?`, `in`, `not in`

Reply:
I added inline comments for this in c30d338